source/adapter.js

```diff
@@ -1036,6 +1040,14 @@ if ( (navigator.mozGetUserMedia ||
         ConstructSessionDescription(info.type, info.sdp);
     };
 
+    MediaStream = function (mediaStreamOrTracks) {
+      AdapterJS.WebRTCPlugin.WaitForPluginReady();
+      if (typeof mediaStreamOrTracks !== 'undefined' && mediaStreamOrTracks !== null) {
```

Comment:
@johache yes, we can remove this if statement in javascript.
**Explanation :** In the plugin c++ code, there is an if condition which is true when mediaStreamOrTracks is not available. That if condition is true when mediaStreamOrTracks javascript object is either null or undefined. In that case, a new MediaStream object will be created without any streams in it (as if it were created without any arguments for MediaStream() constructor).